### PR TITLE
Portfolio setup and mobile first project review

### PR DESCRIPTION
**Pull request by akezeth**
Added linter configuration.
Added header and headline sections in 'index.html' file.
Added appropriate styling according to the Figma design template.

### CODE REVIEW

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @akezeth 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ figma design followed

Required Changes ♻️
Check the comments under the review
tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by akezeth** [comment]
Added padding for the 'I'm a Software Developer' heading.
Added background image.
Changed the list-style-type to none.

File: style.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  line-height: 24px;
+  padding-top: 12px;
+}
+
+.socialmedia {
+  width: 200px;
+  height: 40px;
+  display: flex;
+  padding-top: 30px;
+  justify-content: space-around;
```
[the remark this answers is not in the document]
Comment (OyePriscilla) [reply style.css:82]: Awesome, I could observed that you've implemented this generally for `ul` in line 7. 👍

**Review by OyePriscilla — APPROVED** [review]
Hi @akezeth 👏 👏 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Use flexbox to place elements in the page ✔️ 
Use images and backgrounds to enhance the look of the website ✔️ 
Made required changes ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**